charts/velocity-operator/values.yaml

```diff
@@ -62,6 +62,9 @@ velocity:
 tags:
   cloudResources: false   # All cloud resources will be enabled if this is true
 
+registry:
+  enabled: false # registry will be enabled if this is true
```

Comment:
This is something the customer's will read so if u can explain what is registry in a few words that would be great